### ISSUE TITLE
[Blog post] When to underline links and when not

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#112]
After a long discussion in the Dutch WordPress Slack, we discovered that there is much misunderstanding about when to underline links and when not. And is it even required by WCAG?

The reason this discussion came up was because of the [T-shirt](https://webbed-briefs.teemill.com/product/underline-your-links/) with the text `Underline your f*cking links, you sociopaths`, by Heydon Pickering.

So, we need a blogpost describing when to use underlining (or other means of distinguishing) a link in pieces text.
And also when not (link in a menu). 

We can reference this post later with the topic [Link texts](https://wpaccessibility.org/docs/topics/content/links/link-text/).

Related WCAG Success Criterion: 
- [1.4.1 Use of color](https://www.w3.org/WAI/WCAG22/quickref/#use-of-color)
- [1.4.11 Non-text Contrast](https://www.w3.org/WAI/WCAG22/quickref/#non-text-contrast)
- [2.4.7 Focus Visible](https://www.w3.org/WAI/WCAG22/quickref/#focus-visible)

Resources:
- https://webaim.org/techniques/hypertext/link_text#underlining
- and any more you can find.
